Shared/Samples/Set visibility of subtype sublayer/SetVisibilityOfSubtypeSublayerView.Model.swift

```diff
@@ -0,0 +1,164 @@
+        private let featureTable = ServiceFeatureTable(url: .featureServiceURL)
+        
+        /// A subtype feature layer created from the service feature table.
+        private let subtypeFeatureLayer: SubtypeFeatureLayer
+        
+        /// The subtype sublayer of the subtype feature layer in this sample.
+        private var subtypeSublayer: SubtypeSublayer?
+        
+        /// The renderer of the subtype feature layer.
+        private var originalRenderer: Renderer?
+        
+        /// The  subtype sublayer's label definition.
+        private(set) var labelDefinition: LabelDefinition = {
+            // Make and stylize the text symbol.
+            let textSymbol = TextSymbol(color: .blue, size: 10.5)
+            textSymbol.backgroundColor = .clear
+            textSymbol.outlineColor = .white
+            textSymbol.haloColor = .white
+            textSymbol.haloWidth = 2
+            // Make a label definition and adjust its properties.
+            let labelExpression = SimpleLabelExpression(simpleExpression: "[nominalvoltage]")
+            let labelDefinition = LabelDefinition(labelExpression: labelExpression, textSymbol: textSymbol)
+            labelDefinition.placement = .pointAboveRight
+            labelDefinition.usesCodedValues = true
+            return labelDefinition
+        }()
+        
+        // The formatter used to generate strings from scale values.
+        private let scaleFormatter: NumberFormatter = {
+            let numberFormatter = NumberFormatter()
+            numberFormatter.numberStyle = .decimal
+            numberFormatter.maximumFractionDigits = 0
+            return numberFormatter
+        }()
+        
+        /// A Boolean value indicating whether the settings should be presented.
+        @Published var isShowingSettings = false
+        
+        /// A Boolean value indicating whether to show the subtype sublayer.
+        @Published var showsSublayer = true
+        
+        /// A Boolean value indicating whether to show the subtype sublayer's renderer.
+        @Published var showsOriginalRenderer = true
+        
+        /// The current scale of the map.
+        var currentScale: Double = .zero
+        
+        /// The map's current scale value in text.
+        @Published var currentScaleText: String = ""
+        
+        /// The subtype sublayer's minimum scale value in text.
+        @Published private(set) var minimumScaleText: String = "Not Set"
+        
+        init() {
+            map.initialViewpoint = .initialViewpoint
+            subtypeFeatureLayer = SubtypeFeatureLayer(featureTable: featureTable)
+            subtypeFeatureLayer.scalesSymbols = false
+        }
+        
+        deinit {
+            ArcGISEnvironment.authenticationManager.arcGISCredentialStore.removeAll()
+        }
+        
+        /// Performs important tasks including adding credentials, loading and adding operational layers.
+        @MainActor
```

Comment:
Phil [suggested](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/199#discussion_r1232571109) that the observable objects should have the `@MainActor` attribute added to the type, instead of certain methods, according to Apple.